libs2eplugins/src/s2e/Plugins/Core/BaseInstructions.cpp

```diff
@@ -723,42 +723,19 @@ void BaseInstructions::getConstraintsCountForExpression(S2EExecutionState *state
  */
 void BaseInstructions::forkCount(S2EExecutionState *state) {
     target_ulong count;
-    target_ulong nameptr;
 
     state->jumpToSymbolicCpp();
 
     state->regs()->read(CPU_OFFSET(regs[R_EAX]), &count, sizeof count);
-    state->regs()->read(CPU_OFFSET(regs[R_ECX]), &nameptr, sizeof nameptr);
-
-    std::string name;
-
-    if (!state->mem()->readString(nameptr, name)) {
-        getWarningsStream(state) << "Could not read string at address " << hexval(nameptr) << "\n";
-
-        state->regs()->write<target_ulong>(CPU_OFFSET(regs[R_EAX]), -1);
-        return;
-    }
-
-    klee::ref<klee::Expr> var = state->createSymbolicValue<uint32_t>(name, 0);
-
-    state->regs()->write(CPU_OFFSET(regs[R_EAX]), var);
     state->regs()->write<target_ulong>(CPU_OFFSET(eip), state->regs()->getPc() + 10);
```

Comment:
Maybe lcov didn't reach 100% yesterday because I forgot to do this?

Reply:
It's possible, not sure.